Assets/MRTK/SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -260,8 +260,8 @@ private void UpdateHitTarget()
 
                     if (isHit)
                     {
-                        LookedAtTarget = hitInfo.collider.gameObject;
-                        LookedAtEyeTarget = LookedAtTarget.GetComponent<EyeTrackingTarget>();
+                        LookedAtEyeTarget = hitInfo.collider.transform.GetComponent<EyeTrackingTarget>();
+                        LookedAtTarget = LookedAtEyeTarget?.gameObject;
```

Comment:
I think EyeTrackingTarget is a MonoBehaviour, so we probably want an explicit `LookedAtEyeTarget != null` check here instead of `?.`